service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionController.java

```diff
@@ -75,18 +76,16 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToStartEvent(
 
         if (errors.isEmpty()) {
             try {
-                String label = String.format(CONSENT_TEMPLATE, userDetailsService.getUserName(authorization));
-                data.put("submissionConsentLabel", label);
-
                 if (featureToggleService.isFeesEnabled()) {
                     FeesData feesData = feeService.getFeesDataForOrders(caseData.getOrders());
                     data.put("amountToPay", BigDecimalHelper.toCCDMoneyGBP(feesData.getTotalAmount()));
+                    data.put("displayAmountToPay", YES.getValue());
                 }
-
             } catch (FeeRegisterException ignore) {
-                // TODO: 21/02/2020 Replace me in FPLA-1353
-                //  this is an error message for when the Fee Register is unavailable
-                errors.add("XXX");
+                data.put("displayAmountToPay", NO.getValue());
+            } finally {
+                String label = String.format(CONSENT_TEMPLATE, userDetailsService.getUserName(authorization));
```

Comment:
personally i would move this before try-catch instead of using finally, but it is ok